django/contrib/staticfiles/storage.py

```diff
@@ -403,10 +440,10 @@ def post_process(self, *args, **kwargs):
 
     def save_manifest(self):
         payload = {'paths': self.hashed_files, 'version': self.manifest_version}
-        if self.exists(self.manifest_name):
-            self.delete(self.manifest_name)
+        if self.manifest_handler.exists(self.manifest_name):
+            self.manifest_handler.delete(self.manifest_name)
         contents = json.dumps(payload).encode()
```

Comment:
Maybe change this to `contents = json.dumps(payload, sort_keys=True, indent=4).encode()` ?

Reply:
Fixed